api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -268,6 +268,8 @@ private User initializeUserIfNeeded() {
     BillingProjectStatus status;
     try {
       status = fireCloudService.getBillingProjectMemberships().stream()
+          .filter(m -> m.getProjectName() != null)
+          .filter(m -> m.getCreationStatus() != null)
```

Comment:
Out of curiosity, can FC actually return null for this value? (fine to check defensively for it either way)

What line of this PR did that NPE effectively come from? While you're here, perhaps check for `fcToWorkbenchBillingMap.containsKey()` as well?

Reply:
I don't know if FC can return a null here - this is more of a defensive attempt than root cause analysis. The associated test also triggers the error case seen in stackdriver modulo the filters.

The error is triggered from the `.findFirst()` call. If the previous map function returns a null, this stream call fails.

Reply:
And 👍 on the `fcToWorkbenchBillingMap` idea.